src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/FrameworkReferenceProjectBuild.xaml

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Rule Name="FrameworkReferenceProjectBuild"
+      PageTemplate="generic"
+      xmlns="http://schemas.microsoft.com/build/2009/properties">
+
+  <!-- Represents the set of <FrameworkReference /> items that are gathered during a design-time build to be pushed to Solution Restore service -->
+  
+  <Rule.DataSource>
+    <DataSource HasConfigurationCondition="False"
+                ItemType="FrameworkReference"
+                MSBuildTarget="CollectFrameworkReferences"
+                Persistence="ProjectFile"
+                SourceOfDefaultValue="AfterContext"
+                SourceType="TargetResults" />
+  </Rule.DataSource>
+
+  <StringProperty Name="PrivateAssets" />
```

Comment:
@nkolev92 I've already added PrivateAssets and will automatically pass it to NuGet when requested.